src/main/java/jp/co/future/uroborosql/SqlAgentImpl.java

```diff
@@ -1361,7 +1359,24 @@ public void close() throws SQLException {
 				}
 			}
 		}
+	}
 
+	/**
+	 * 経過時間を計算し、HH:MM:SS.SSSにフォーマットする.
+	 *
+	 * @param start 開始時間
+	 * @param end 終了時間
+	 * @return フォーマットした経過時間
+	 */
+	private String formatElapsedTime(final Instant start, final Instant end) {
+		Duration duration = Duration.between(start, end);
+		long elapsed = duration.getSeconds();
+		long hours = elapsed / 3_600;
+		long minutes = elapsed % 3_600 / 60;
+		long seconds = elapsed % 60;
+		long micros = (duration.toNanos() - elapsed * 1_000_000_000) / 1_000;
+
+		return String.format("%02d:%02d:%02d.%06d", hours, minutes, seconds, micros);
```

Comment:
@HidekiSugimoto189 

I think the Java Date And Time API seems to be better implemented as follows.

```java
private String formatElapsedTime(final Instant start, final Instant end) {
    Duration duration = Duration.between(start, end);
    LocalTime durationTime = LocalTime.MIDNIGHT.plus(duration);
    return DateTimeFormatter.ofPattern("HH:mm:ss.SSS").format(durationTime);
}
```

Reply:
@shout-star 

Thanks review.

fixed code.

```java
private static final DateTimeFormatter ELAPSED_TIME_FORMAT = DateTimeFormatter.ofPattern("HH:mm:ss.SSSSSS");

private static String formatElapsedTime(final Instant start, final Instant end) {
	return ELAPSED_TIME_FORMAT.format(LocalTime.MIDNIGHT.plus(Duration.between(start, end)));
}
```